docs/source/user_guide/runtime-image-conf.md

```diff
@@ -189,16 +189,23 @@ The name and tag of an existing container image in a container registry that mee
 
```

Comment:
Here, you could specify instead:

The name and tag or name and SHA256-digest of an existing container image in a container registry that meets the stated prerequisites. Use SHA256-digest format in airgapped Openshift environments to support ImageContentSourcePolicy and automatic referral of images to a target repository server or to just support unique non-modifyable image digests instead of potentially transient tags. SHA256-digest format is supported on all platforms for image references. This property is required.